src/main/webapp/app/core/user/settings/passkey-settings/entities/malformed-bitwarden-credential.ts

```diff
@@ -1,53 +0,0 @@
-/**
- * <a href="https://chromewebstore.google.com/detail/nngceckbapebfimnlniiiahkandclblb?utm_source=item-share-cb">Bitwarden Browser Plugin</a>
- * does not return a valid Credential object on Chrome https://github.com/bitwarden/clients/issues/12060
- */
-export interface MalformedBitwardenCredential {
-    id: string;
-    rawId: Record<string, number>;
-    type: string;
-    authenticatorAttachment: string;
```

Comment:
Now split into `src/main/webapp/app/core/user/settings/passkey-settings/entities/malformed-bitwarden-login-credential.ts` and `src/main/webapp/app/core/user/settings/passkey-settings/entities/malformed-bitwarden-registration-credential.ts`